ts/localredirectiondebugger/v1.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Supports working with the Debug (Redirect locally) command.
+ */
+export interface LocalRedirectionDebuggerV1 {
+    /**
+     * Registers a local redirection debugger. The debugger will be invoked on user action
+     * when the user selects the Debug (Redirect locally) command in the cluster explorer,
+     * or when running the Debug (Redirect locally) command in the command palette. The
+     * local redirection debugger wil show its own user interface once invoked.
+     * @param localRedirectionDebugger The local redirection debugger to be registered.
+     */
+    register(localRedirectionDebugger: LocalRedirectionDebuggerV1.LocalRedirectionDebugger): void;
+    /**
+     * Lists registered local redirection debuggers.
+     * @returns The local redirection debuggers registered for the Debug (Redirect locally) command.
+     */
+    list(): ReadonlyArray<LocalRedirectionDebuggerV1.LocalRedirectionDebugger>;
```

Comment:
What's the need of having a `list` function here? The API will be mainly used by other extensions that want to extend/use the local debugger in k8s-tools, so in which case one extender would want to check which are all the local redirection debuggers registered in k8s-tools? Wouldn't it be better to have a `find(id)` so if you need to use a specific local debugger you can look directly for it?

Reply:
I added this for consistency with the cluster provider API that offers a similar `list` method. That said, I agree this doesn't sound required right now. I can't think of a case where a consumer of this API would actually need to get this, so I'm going to remove this method for now.